ui/src/shared/actions/v2/queryBuilder.ts

```diff
@@ -202,12 +204,11 @@ export const loadBuckets = () => async (
   dispatch: Dispatch<Action>,
   getState: GetState
 ) => {
-  const queryURL = getActiveQuerySource(getState()).links.query
-
   dispatch(setBuilderBucketsStatus(RemoteDataState.Loading))
 
   try {
-    const buckets = await fetcher.findBuckets(queryURL)
+    const {data} = await bucketsAPI.bucketsGet('')
```

Comment:
do you need to pass the empty string in?

Reply:
yeah, because we don't want any specific org, rather we want any buckets you have access to